app/views/cdo/edit/exemption-details.njk

```diff
@@ -0,0 +1,80 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+
+{% extends '_layout.njk' %}
+
+{% block pageTitle %}{{ serviceName }}{% endblock %}
+
+{% block beforeContent %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-half">
+      {{ govukBackLink({
+        text: "Back",
+        href: model.backLink,
+        attributes: {
+          id: "back"
+        }
+        }) 
+      }}
+    </div>
+  </div>
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    {% if model.errors | length > 0 %}
+      {{ govukErrorSummary({
+        titleText: "There is a problem",
+        errorList: model.errors
+      }) }}
+    {% endif %}
+
+    <h2 class="govuk-heading-m">Owner details</h2>
+
+    <form method="POST" action="{{ model.formAction }}" novalidate autocomplete="off">  
+      {{ govukSelect(model.exemption.status) }}
+
+      {{ govukDateInput(model.exemption.certificateIssued) }}
+
+      {{ govukDateInput(model.exemption.cdoIssued) }}
+
+      {{ govukDateInput(model.exemption.cdoExpiry) }}
+
+      {{ govukSelect(model.exemption.court) }}
```

Comment:
Was court to be the autocomplete? Probably not necessary as mostly would have been completed using autocomplete when creating the CDO